example/lib/home_page.dart

```diff
@@ -410,15 +410,15 @@ ButtonStyle buttonStyle(BuildContext context) {
         return Web3ModalTheme.getDataOf(context).textStyles.small600.copyWith(
               color: (states.contains(MaterialState.disabled))
                   ? Web3ModalTheme.colorsOf(context).foreground300
-                  : Web3ModalTheme.colorsOf(context).background100,
+                  : Web3ModalTheme.colorsOf(context).inverse100,
```

Comment:
does this work for both light and dark mode? inverse100 is always white

Reply:
Correcto, but this is just the sample side, and since I have fixed themes I'm ok with that. The developer can change it as they please